Gems/AWSGameLift/Code/AWSGameLiftServer/Include/Request/IAWSGameLiftServerRequests.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/RTTI/BehaviorContext.h>
+#include <AzCore/std/string/string.h>
+#include <AzFramework/Session/ISessionRequests.h>
+
+namespace AWSGameLift
+{
+    //! IAWSGameLiftServerRequests
+    //! Server interfaces to expose Amazon GameLift Server SDK
+    class IAWSGameLiftServerRequests
```

Comment:
Code style "We don't recommend using 'C','T','I' or any other letter to prefix a class"

Reply:
I am following the networking code style, where do you see that we should not have "I" to prefix a class?